gensrc/thrift/CloudConfiguration.thrift

```diff
@@ -23,12 +23,7 @@ enum TCloudType {
     ALIYUN
 }
 
-struct TCloudProperty {
-    1: required string key;
-    2: required string value;
-}
-
 struct TCloudConfiguration {
     1: optional TCloudType cloud_type;
-    2: optional list<TCloudProperty> cloud_properties;
```

Comment:
This change will cause compatibility issues.
The process who is running new binary can not recognize the older version's requests.

Reply:
Done, I have tested it using old fe and new be.